packages/ui/src/Text/index.tsx

```diff
@@ -135,6 +138,9 @@ const VARIANT_MAP: Record<TextVariant, { element: ElementType; classes: string }
   detailTechnical: { element: 'span', classes: detailTechnical },
   technical: { element: 'span', classes: technical },
   body: { element: 'span', classes: body },
+  tableHeading: { element: 'span', classes: tableHeading },
+  tableHeadingMedium: { element: 'span', classes: tableHeadingMedium },
+  tableHeadingSmall: { element: 'span', classes: tableHeadingSmall },
```

Comment:
suggestion: add `tableItem`, `tableItemMedium`, and `tableItemSmall` too. You would need it for several cells in the positions table

<img width="591" alt="image" src="https://github.com/user-attachments/assets/dfd1aeea-8330-4d1a-94d8-c7f53e824f78" />